src/main/webapp/assets/modules/loading-box/loadingBoxController.js

```diff
@@ -1,4 +1,4 @@
-techlooper.controller("loadingBoxController", function (utils, jsonValue, $scope, localStorageService, $location) {
+techlooper.controller("loadingBoxController", function (utils, jsonValue, $scope, localStorageService, $location, utils) {
```

Comment:
Line is too long.<br>'utils' is already defined.<br>'techlooper' is not defined.